Strategy11/formidable-forms
# Make block title and descriptions translated in PHP

Fix https://github.com/Strategy11/formidable-pro/issues/5056

<img width="728" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/41271840/8e0513a9-c8ad-4bdf-955b-a61ef0236d25">

### Test steps

1. Change your site language to something other than English
2. Try adding the four blocks (Formidable Forms, Formidable Forms Modal, Formidable Chart, Calculator Form) into a page 
3. Inspect each block to check the block title and descriptions are translated based on the availability of translations for the language.
